roles/openshift_examples/tasks/stream_secrets.yml

```diff
@@ -31,4 +31,10 @@
 - name: Modify registry paths if registry_url is not registry.redhat.io
   shell: >
     find {{ examples_base }} -type f | xargs -n 1 sed -i 's|registry.redhat.io|{{ registry_host | quote }}|g'
-  when: registry_host != '' and openshift_examples_modify_imagestreams | default(False) | bool
+  when: openshift_examples_registryurl.split('/') | length > 2 and registry_host != '' and openshift_examples_modify_imagestreams | default(False) | bool
```

Comment:
Please use multi-line 'and' conditionals:
```yaml
  when:
    - openshift_examples_registryurl.split('/') | length > 2
    - registry_host != ''
    - openshift_examples_modify_imagestreams | default(False) | bool
```

Reply:
Thank you for your kind review, I've modified as you mentioned.